### ISSUE TITLE
Swap sponsorship tickets with tokens (StakePrizePool)

### ISSUE THREAD

**Issue by moonery-dev** [pooltogether/pooltogether-pool-contracts#296]
How to swap my sponsorship token back to MY ERC20 token? 

I tried Function DepositTo and using the sponsorship token as tokencontroller.

**Comment by asselstine** [comment]
Could you rephrase your question?  It's not very clear to me.

I'm guessing you're trying to swap the sponsorship token back for the underlying asset?

In that case you need to use [withdrawInstantlyFrom](https://github.com/pooltogether/pooltogether-pool-contracts/blob/f64f6e0e4ed92db8f76539388b834d85e4e8d1f4/contracts/prize-pool/PrizePool.sol#L319)
